spec/support/factories/votes.rb

```diff
@@ -0,0 +1,7 @@
+FactoryBot.define do
+  factory :vote do
+    value 1
```

Comment:
could be cool to randomize the values here with `[1, 0, -1].sample` and setup traits for `:upvote`, `:downvote` and `:novote`

Reply:
I think I get where you're going with this but I don't know exactly what you mean on a detailed level. Since I don't want to use zero in my validations (there's a null constraint on the value attribute at the db table level), I'm only dealing with either 1 or -1. I created traits for up_vote and down_vote, which may be useful for future feature specs.

Reply:
FWIW, `0` is not null. I also pulled those 3 numbers because that's what the numerical validation implied before the changes. `[1, -1].sample` would be the correct update now that I see all the values that `value` can be. I just thought it'd be cool to still be able to say `build :vote` if you didn't care what the `value` was. The traits look great 👍

Reply:
Gotcha - I'm making an update now to say:
`FactoryBot.define do`
  `    factory :vote do`
    `value [1, -1].sample`